documentation/release_notes.rst

```diff
@@ -8,16 +8,53 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added a new implementation for ``par`` and ``par_unseq`` execution policies based on OpenMP* 4.5 pragmas.
+  It can be enabled with the ``ONEDPL_USE_OPENMP_BACKEND`` macro. For more details, see `the Macros`_ page.
+- Added the range-based version of the ``reduce_by_segment`` algorithm and improved performance of
+  the iterator-based ``reduce_by_segment`` APIs. 
```

Comment:
```suggestion
- Added the range-based variation of the ``reduce_by_segment`` algorithm and improved performance of
  the iterator-based ``reduce_by_segment`` variations. 
```

Reply:
In previous release notes we used "version" for range-based algorithms. Does this change apply only to this note? If it should be added retrospectively, I'd prefer keeping "version" for consistency.

Reply:
Let's keep it then, it's not critical